### DISCUSSION TITLE
Landing page

### DISCUSSION THREAD

**Discussion by Ahmadnew** [dropways/deskapp#55] [General]
Please Release Landing PAge Soon. Please Tell Me Date Of Releasing . If Anyone Have Landing Page Like This Template Please Send me

**Comment by dropways** [comment]
Hello @Ahmadnew 

 I am planning to work on landing page meanwhile you can see my other landing page if you ok with this landing page you can use it
 
 **GitHub :-** https://github.com/dropways/Real-Estate-Property-Landing-Webpage
 
 **Demo URL :-** https://dropways.github.io/Real-Estate-Property-Landing-Webpage/
 
 Thank you.
